portality/scripts/article_cleanup_sync.py

```diff
@@ -0,0 +1,137 @@
+"""
+For each article in the DOAJ index:
+    * Checks that it has a corresponding journal, deletes it otherwise (if -d set)
```

Comment:
oops, I did change that. I changed it to -w which handles all writes and deletes. I was wondering, however,  as I wrote the instructions above whether we might want to write updates but not do deletes. Should I put the -d flag back in?